### PR TITLE
fix (DashboardSidebarListItem): #49 - use Next Link cmpt instead of Mui Link cmpt

### PR DESCRIPTION
**Pull request by f-jost**
# Description

In preprod, navigate on the admin part, google chrome asks user to authenticate every time when clicking on the sidebar links.

1 - short refacto in _app.tsx and Layout.tsx
2 - Link component from Mui was used instead of Next Link component, which caused a refresh on the application clicking on the links. Using the correct component, which helps mitigate chrome's basic authentication request,

This does not solve the whole problem because sometimes the problem randomly reoccurs on any link that use the good component..

# Changes

| Q             | A        
|---------------| ---------
| Issue         | #49 
| Type          | <ul><li>- [ ] Feat</li><li>- [ ] Hotfix</li><li>- [ ] Doc</li><li>- [x] Refacto</li><li> [x] Fix</ul>
| Break changes |  No

### CODE REVIEW

File: pwa/pages/_app.tsx [hunk 1]
```diff
@@ -2,223 +2,182 @@ import Layout from '@components/common/Layout';
...
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-640-1136.jpg"
+          media="(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"></link>
+        <link
+          rel="apple-touch-startup-image"
+          href="/pwa_icons/apple-splash-1136-640.jpg"
+          media="(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"></link>
+      </Head>
+      <GoogleOAuthProvider
```
Comment (davy-beauzil) [inline pwa/pages/_app.tsx:171]: Why only this context provider has not moved in Layout.tsx ?

File: pwa/components/common/Layout.tsx [hunk 2]
```diff
@@ -17,7 +33,19 @@ const Layout = ({
 
   return (
     <QueryClientProvider client={queryClient}>
-      <Hydrate state={dehydratedState}>{children}</Hydrate>
+      <Hydrate state={dehydratedState}>
+        <AuthProvider>
+          <ThemeProvider theme={theme}>
+            <CssBaseline />
```
Comment (davy-beauzil) [inline pwa/components/common/Layout.tsx:39]: I don't see breaking changes in the front, but before, this component had other components as children, but not now.
It's normal ?

**Review by davy-beauzil — CHANGES_REQUESTED** [review]
These are 2 files in which the mui Link component is still in use :
- https://github.com/RustineLibre/RustineLibre/blob/2c0c8dd55dd4aa7d9dd2659533d921e6856bd302/pwa/components/dashboard/agenda/ModalShowAppointment.tsx#L40
- https://github.com/RustineLibre/RustineLibre/blob/2c0c8dd55dd4aa7d9dd2659533d921e6856bd302/pwa/pages/login.tsx#L12

File: pwa/components/common/Layout.tsx (hunk 2 above)
Comment (f-jost) [reply pwa/components/common/Layout.tsx:39]: Yes, it's a self closing component, see: https://mui.com/material-ui/react-css-baseline/
There is another component for wrapping children: ScopedCssBaseline - https://mui.com/material-ui/react-css-baseline/#scoping-on-children

File: pwa/pages/_app.tsx (hunk 1 above)
Comment (f-jost) [reply pwa/pages/_app.tsx:171]: I prefer to let this context on top of the app with the <Analytics /> too for more clarity

**Review by davy-beauzil — APPROVED** [review]

**Pull request merged**